src/_P088_HeatpumpIR.ino

```diff
@@ -43,38 +43,40 @@
  *
  */
 
- #include <FujitsuHeatpumpIR.h>
- #include <PanasonicCKPHeatpumpIR.h>
- #include <PanasonicHeatpumpIR.h>
- #include <CarrierHeatpumpIR.h>
- #include <MideaHeatpumpIR.h>
- #include <MitsubishiHeatpumpIR.h>
- #include <SamsungHeatpumpIR.h>
- #include <SharpHeatpumpIR.h>
- #include <DaikinHeatpumpIR.h>
- #include <MitsubishiHeavyHeatpumpIR.h>
- #include <MitsubishiSEZKDXXHeatpumpIR.h>
- #include <HyundaiHeatpumpIR.h>
- #include <HisenseHeatpumpIR.h>
- #include <GreeHeatpumpIR.h>
- #include <FuegoHeatpumpIR.h>
- #include <ToshibaHeatpumpIR.h>
- #include <ToshibaDaiseikaiHeatpumpIR.h>
- #include <IVTHeatpumpIR.h>
- #include <HitachiHeatpumpIR.h>
- #include <BalluHeatpumpIR.h>
- #include <AUXHeatpumpIR.h>
+#include <FujitsuHeatpumpIR.h>
+#include <PanasonicCKPHeatpumpIR.h>
+#include <PanasonicHeatpumpIR.h>
+#include <CarrierHeatpumpIR.h>
+#include <MideaHeatpumpIR.h>
+#include <MitsubishiHeatpumpIR.h>
+#include <SamsungHeatpumpIR.h>
+#include <SharpHeatpumpIR.h>
+#include <DaikinHeatpumpIR.h>
+#include <MitsubishiHeavyHeatpumpIR.h>
+#include <MitsubishiSEZKDXXHeatpumpIR.h>
+#include <MitsubishiMSCHeatpumpIR.h>
+#include <MitsubishiHeavyFDTCHeatpumpIR.h>
+#include <HyundaiHeatpumpIR.h>
+#include <HisenseHeatpumpIR.h>
+#include <GreeHeatpumpIR.h>
+#include <FuegoHeatpumpIR.h>
+#include <ToshibaHeatpumpIR.h>
+#include <ToshibaDaiseikaiHeatpumpIR.h>
+#include <IVTHeatpumpIR.h>
+#include <HitachiHeatpumpIR.h>
+#include <BalluHeatpumpIR.h>
+#include <AUXHeatpumpIR.h>
 
 // Array with all supported heatpumps
 HeatpumpIR *heatpumpIR[] = {new PanasonicCKPHeatpumpIR(), new PanasonicDKEHeatpumpIR(), new PanasonicJKEHeatpumpIR(),
                             new PanasonicNKEHeatpumpIR(), new PanasonicLKEHeatpumpIR(),
                             new CarrierNQVHeatpumpIR(), new CarrierMCAHeatpumpIR(),
                             new MideaHeatpumpIR(), new FujitsuHeatpumpIR(),
                             new MitsubishiFDHeatpumpIR(), new MitsubishiFEHeatpumpIR(), new MitsubishiMSYHeatpumpIR(), new MitsubishiFAHeatpumpIR(),
-							new MitsubishiKJHeatpumpIR(),
+                            new MitsubishiKJHeatpumpIR(), new MitsubishiHeavyFDTCHeatpumpIR(),
                             new SamsungAQVHeatpumpIR(), new SamsungFJMHeatpumpIR(),new SharpHeatpumpIR(), new DaikinHeatpumpIR(),
                             new MitsubishiHeavyZJHeatpumpIR(), new MitsubishiHeavyZMHeatpumpIR(),
-                            new MitsubishiSEZKDXXHeatpumpIR(),
+                            new MitsubishiSEZKDXXHeatpumpIR(), new MitsubishiMSCHeatpumpIR(),
                             new HyundaiHeatpumpIR(), new HisenseHeatpumpIR(),
                             new GreeGenericHeatpumpIR(), new GreeYANHeatpumpIR(), new GreeYAAHeatpumpIR(),
                             new FuegoHeatpumpIR(), new ToshibaHeatpumpIR(), new ToshibaDaiseikaiHeatpumpIR(),
```

Comment:
Just as a generic remark and doesn't have to be fixed right now, but may need a fix in another PR.
This array is creating all (!!) supported models in an array.
This means all data that each class allocates as static or as member data will be taking resources on the heap.
This is even when P088 is not selected or enabled.

I made a new issue for it: https://github.com/letscontrolit/ESPEasy/issues/2884

The best way for this is to use the "factory" design pattern to create the object when needed.
These classes all inherit from the `HeatpumpIR` class, so this factory can just return a pointer of this type and when using it, you can cast it to the right type.

Another bug in this is that the base class does not have a virtual destructor.
Right now, we don't destruct the objects, but if you make a factory for it, you should also be able to destruct it and thus you need to have a virtual destructor as you will call the delete on the base class pointer.